### PR TITLE
migrate to junit5

### PR DESCRIPTION
**Pull request by brig**

### CODE REVIEW

File: dependency-manager/src/test/java/com/walmartlabs/concord/dependencymanager/DependencyManagerTest.java_ [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.junit.Assert.assertEquals;
+
+@Ignore
+public class DependencyManagerTest {
```
Comment (benbroadaway) [inline dependency-manager/src/test/java/com/walmartlabs/concord/dependencymanager/DependencyManagerTest.java_:58]: This file can be removed right?

Comment (brig) [reply dependency-manager/src/test/java/com/walmartlabs/concord/dependencymanager/DependencyManagerTest.java_:58]: ouch. thanks

**Comment by brig** [comment]
@concord-ci test it

**Comment by brig** [comment]
@concord-ci test it

**Review by ibodrov — DISMISSED** [review]

**Comment by ibodrov** [comment]
@concord-ci test it

**Comment by brig** [comment]
@concord-ci test it

**Review by ibodrov — APPROVED** [review]

**Pull request merged**